docs/man/git-lfs-ls-files.1.ronn

```diff
@@ -3,12 +3,15 @@ git-lfs-ls-files(1) -- Show information about Git LFS files in the index and wor
 
 ## SYNOPSIS
 
-`git lfs ls-files` [<ref>]
+`git lfs ls-files` [<ref>]<br>
+`git lfs ls-files` <ref> <ref>
 
 ## DESCRIPTION
 
 Display paths of Git LFS files that are found in the tree at the given
 reference.  If no reference is given, scan the currently checked-out branch.
+If two references are given, the LFS files that are modified between the two
+references are shown.
```

Comment:
It looks like your test checks that we don't print deletions. That's fine, but we probably want to mention it here, since Git considers a deletion to be a modification. Maybe add a semicolon and “deletions are not listed”?

Reply:
Sounds good! What do you prefer, that I create a new commit or amend the original commit?

Reply:
If you could amend the original commit, that would be great.